.github/workflows/release-fleet.yml

```diff
@@ -91,19 +91,22 @@ jobs:
           repo: "ghcr.io/rancher/fleet"
           tag: "${GITHUB_REF_NAME}"
         run: |
+          # Replace rancher/fleet, rancher/fleet-agent and rancher/gitjob image names, but not eg. rancher/kubectl
           sed -i \
-              -e "s@repository:.*@repository: $repo@" \
-              -e "s/tag:.*/tag: ${VERSION}/" \
+              -e "s@repository: rancher/\(\(fleet.*\|gitjob\)\).*@repository: ${repo/fleet/\\1}@" \
```

Comment:
```suggestion
              -e "s@repository: rancher/\(fleet.*\|gitjob\).*@repository: $repo/fleet/\\1@" \
```

Reply:
Thanks for spotting the extra pair of parentheses, I'll remove them.
I find the braces still to be needed for string substitution though, omitting them would yield a diff such as:
```
diff --git a/charts/fleet/values.yaml b/charts/fleet/values.yaml
index 8ff51483..ba702362 100644
--- a/charts/fleet/values.yaml
+++ b/charts/fleet/values.yaml
@@ -1,11 +1,11 @@
 image:
-  repository: rancher/fleet
-  tag: dev
+  repository: ghcr.io/rancher/fleet/fleet/fleet
+  tag: v42.0.0
   imagePullPolicy: IfNotPresent
 
 agentImage:
-  repository: rancher/fleet-agent
-  tag: dev
+  repository: ghcr.io/rancher/fleet/fleet/fleet-agent
+  tag: v42.0.0
   imagePullPolicy: IfNotPresent
 
 # For cluster registration the public URL of the Kubernetes API server must be set here
@@ -89,8 +89,8 @@ leaderElection:
   renewDeadline: 25s
 
 gitjob:
-  repository: rancher/gitjob
-  tag: dev
+  repository: ghcr.io/rancher/fleet/fleet/gitjob
+  tag: v42.0.0
   imagePullPolicy: IfNotPresent
   nodeSelector:
     kubernetes.io/os: linux
```

Reply:
I see, the input is `ghcr.io/rancher/fleet`. 
I'd suggest to change that to  `ghcr.io/rancher` and get rid of the bash parameter expansion inside the regex. Seems redundant.

Reply:
Oh that makes sense! Done, thanks.